source/Dgraph/Client/IDgraphClient.cs

```diff
@@ -31,16 +29,30 @@ namespace Dgraph
     /// has been disposed and calls are made.</exception>
     public interface IDgraphClient : IDisposable
     {
+        /// <summary>
+        /// Log in the client to the default namespace (0) using the provided
+        /// credentials. Valid for the duration the client is alive.
+        /// </summary>
+        Task<Result> Login(string user, string password, CallOptions? options = null)
+        {
+            return LoginIntoNamespace(user, password, 0, options);
```

Comment:
Is that a default that can be replaced later in the user program? shouldn't the value 0 be a variable? just curious.

Reply:
Yes, this is the same as in the dgo client. If the user wants to specify a namespace, they can use the `LoginIntoNamespace` method directly.